src/main/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/util/LocalDateExt.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.educationemploymentapi.util
```

Comment:
Class not used, no unit tests, so can be removed

Reply:
removed